### PR TITLE
[HP-112] Disaply Link Icon For Non-Health-Alert Pages

### PR DESCRIPTION
**Pull request by dchukhin**
https://caktus.atlassian.net/browse/HP-112

This pull request adds the link icon and the word 'PAGE' to each card in the search results that is not a health alert page.

### CODE REVIEW

File: apps/hip/templates/includes/search_result_card.html [hunk 1]
```diff
@@ -14,10 +14,17 @@
         </div>
         <div class="level-right">
           {% if result.specific.get_priority_display %}
+            {# Health alert pages show the icon for their priority and their priority #}
```
Comment (Audiosutras) [inline apps/hip/templates/includes/search_result_card.html:17]:
```suggestion
            {# Health alert pages show the icon for their priority  #}
```

Comment (Audiosutras) [reply apps/hip/templates/includes/search_result_card.html:17]: You may have intended `and their priority`; I wasn't certain though.

**Review by Audiosutras — APPROVED** [review]
👍

File: apps/hip/templates/includes/search_result_card.html (hunk 1 above)
Comment (dchukhin) [reply apps/hip/templates/includes/search_result_card.html:17]: I did actually mean to include both (their priority and the icon for their priority). I'll try to reword it.

**Pull request merged**